tasks.py

```diff
@@ -405,3 +427,21 @@ def print_config():
             for line in f.readlines():
                 message(line.lstrip("export ").rstrip("\n"), Level.BODY)
     print()
+
+
+def create_node_config_file(node_id, ip, monetdb_port, rabbitmq_port):
+    with open(DEFAULT_NODE_CONFIG_FILE) as fp:
+        node_config = toml.load(fp)
+
+    node_config["identifier"] = node_id
+    node_config["monetdb"]["ip"] = ip
+    node_config["monetdb"]["port"] = monetdb_port
+    node_config["rabbitmq"]["ip"] = ip
+    node_config["rabbitmq"]["port"] = rabbitmq_port
+
+    Path(CONFIG_FILES_FOLDER).mkdir(parents=True, exist_ok=True)
+    node_config_file = CONFIG_FILES_FOLDER / f"{node_id}.toml"
+    with open(node_config_file, "w+") as fp:
+        toml.dump(node_config, fp)
+
+    return node_config_file
```

Comment:
Variables node_name/node_id, ip and port are passed as input arguments into multiple task/functions. This makes this code very confusing and the chances of passing a wrong value to one of the tasks are very high.
The idea is to have one config file (per node that we want to start) that holds the values of these variables and any other component that needs these variables' values gets it from there.

node_name/node_id is passed as a variable in:
1)`def config(c, ip=None, node_name=None, monetdb_port=None, rabbitmq_port=None ...)`
2)`def start_node(c, ip, node_name, monetdb_port, rabbitmq_port)`
3)`def create_node_config_file(node_id, ip, monetdb_port, rabbitmq_port)`

ip is passed as a variable in:
1)`def config(c, ip=None, node_name=None, monetdb_port=None, rabbitmq_port=None ...)`
2)`def start_node(c, ip, node_name, monetdb_port, rabbitmq_port)`
3)`def create_node_config_file(node_id, ip, monetdb_port, rabbitmq_port)`
   
port is passed as a variable in:
1)`def config(c, ip=None, node_name=None, monetdb_port=None, rabbitmq_port=None ...)`
2)`def start_node(c, ip, node_name, monetdb_port, rabbitmq_port)`
3)`def create_node_config_file(node_id, ip, monetdb_port, rabbitmq_port)`
4)`def start_monetdb(c, port)`
5)`def load_data_into_db(c, port)`
6)`def config_rabbitmq(c, ports)`
7)`def start_rabbitmq(c, port)`

Reply:
Please check my previous response on how the node's configuration is built on the script and let me know if it makes sense now.